### PR TITLE
Migrate the qemu supervisor (and adjacent components) to tracing

### PR DESCRIPTION
**Pull request by lschuermann**
This allows us to debug why the QEMU supervisor fails to start, without looking at the state changes that are pushed to the switchboard.

I'm sure that we'll want to fine-tune the `#[instrument` scopes later on, but this seems to give a decent amount of debug information while not printing out types that would be too large to display in the context of every message.

### CODE REVIEW

File: switchboard/cli/Cargo.toml [hunk 1]
```diff
@@ -27,6 +27,3 @@ chrono = "0.4.38"
 xdg = "2.5.2"
 ssh2 = "0.9.4"
 base64 = "0.22.1"
-
-[dependencies.tml-switchboard]
-path = "../switchboard"
```
Comment (lschuermann) [inline switchboard/cli/Cargo.toml:32]: This change is not related, but we were pulling in a bunch of other dependencies into the CLI through this false dependency on the switchboard. Things work file with this removed.

Comment (lschuermann) [reply switchboard/cli/Cargo.toml:32]: Split out into #45.

**Review by max-cura — APPROVED** [review]

**Review by max-cura — CHANGES_REQUESTED** [review]
Need to regenerate Cargo.lock.

**Review by max-cura — APPROVED** [review]
LGTM.

**Pull request merged**